static/js/components/subject_page/block_container.tsx

```diff
@@ -40,13 +44,18 @@ export function BlockContainer(props: BlockContainerPropType): JSX.Element {
       })
       .join("\n");
   }
+  const rs: ReplacementStrings = {
+    place: props.place.name,
```

Comment:
since place is optional (from the prop type), we should check that `props.place` is defined before trying to access `props.place.name`

Reply:
good call - done.